pragma-sdk/pragma_sdk/common/fetchers/generic_fetchers/deribit.py

```diff
@@ -0,0 +1,304 @@
+import aiohttp
+import time
+
+from typing import Optional, List, Dict, Any, Tuple
+from aiohttp import ClientSession
+from pydantic.dataclasses import dataclass
+from starknet_py.hash.hash_method import HashMethod
+from starknet_py.hash.utils import compute_hash_on_elements
+
+from pragma_sdk.common.logging import get_pragma_sdk_logger
+from pragma_sdk.common.exceptions import PublisherFetchError
+from pragma_sdk.common.types.pair import Pair
+from pragma_sdk.common.types.types import UnixTimestamp
+from pragma_sdk.common.types.currency import Currency
+from pragma_sdk.common.types.entry import Entry, GenericEntry
+from pragma_sdk.common.types.merkle_tree import MerkleTree
+
+from pragma_sdk.common.fetchers.interface import FetcherInterfaceT
+from pragma_sdk.common.utils import str_to_felt
+
+from pragma_sdk.onchain.constants import DERIBIT_MERKLE_FEED_KEY
+from pragma_sdk.onchain.client import PragmaOnChainClient
+from pragma_sdk.onchain.types import Network
+
+logger = get_pragma_sdk_logger()
+
+
+@dataclass
+class DeribitOptionResponse:
```

Comment:
move types to a `types.py`file

Reply:
Mhhhh not sure I agree - imo since they're exclusively used for this fetcher they should not spill over somewhere else.
What I could do is replace `deribit.py` to a module `deribit/` containing the `fetcher.py` and `types.py` files. Thoughts? @EvolveArt

Reply:
yes that's what I meant indeed

Reply:
done